pkgs/development/python-modules/kconfiglib/default.nix

```diff
@@ -14,6 +15,13 @@ buildPythonPackage rec {
     sha256 = "0g690bk789hsry34y4ahvly5c8w8imca90ss4njfqf7m2qicrlmy";
   };
 
+  patches = [
+    (fetchpatch {
+      url = "https://github.com/ulfalizer/Kconfiglib/pull/119/commits/3161fec0b9ff9154dbd952c3481400118fabb744.patch";
```

Comment:
Patches from unmerged PRs should be vendored in case the PR is force-pushed to.

https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#fetching-patches